src/components/checkboxes/template.njk

```diff
@@ -31,6 +31,10 @@
     legendHtml: params.fieldset.legendHtml,
     legendHintText: params.fieldset.legendHintText,
     legendHintHtml: params.fieldset.legendHintHtml,
+    error: {
+      active: params.fieldset.error.active,
+      text: params.fieldset.error.text
+    },
     errorMessage: params.errorMessage
```

Comment:
This is no longer being used in the fieldset component - if we decide that moving the text / html arguments to error _is_ the right approach, this should be removed.